README.md

```diff
@@ -10,6 +10,19 @@ In addition, the migration files can be written in Groovy in order to run script
 Only requirement is that migration files follow the pattern `V99__Some_name.[sql|groovy]`, where 
 `99` can be any integer value.
 
+### Fixed and amended files
+
+If you have broken migration scripts, you can use the migration tool to add fixed and amended files to existing migration files. These two files are always created together.
+
+1. If the broken file has already been applied, a fixed file will be used. The fixed file name should follow this pattern:
+   `V99_file_to_fix.fixed.[sql|groovy]`
+
+2. If the broken file hasn't been applied, an amended file will be used instead of the original file. The amended file name should follow this pattern:
+   `V99_file_to_fix.amended.[sql|groovy]`
```

Comment:
There can be more than one pair of `fixed` and `amended` files?

Reply:
No, I added that to the Warning below.

Reply:
I fear it may be needed, but it can be kept as a future extension. 


Apart this, I think it would be a bit more clear renaming `.amended` -> `.override` (amend sounds like a minor or incremental change) and `.fixed` -> `.patch`